SPECS/rsyslog/rsyslog.spec

```diff
@@ -128,6 +129,10 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %check
 %make_build check
 
+%pre
+groupadd --system syslog
+useradd --system --comment 'System Logging'  --gid syslog --shell /bin/false syslog
+
```

Comment:
Do we not want an equivalent `%postun` section to remove the user/group on package uninstall?

Reply:
change as suggested. Add extra commands to delete user and group